yarn.lock

```diff
@@ -4298,26 +4298,26 @@
     react-router-dom "6.0.0-beta.0"
     react-use "^15.3.6"
 
-"@roadiehq/backstage-plugin-github-insights@^1.0.0":
-  version "1.0.0"
-  resolved "https://registry.npmjs.org/@roadiehq/backstage-plugin-github-insights/-/backstage-plugin-github-insights-1.0.0.tgz#8b30bd3c4979aab8cd145c4c0b28488badd8e662"
-  integrity sha512-KynUXVGb+lpMoDoGrolOtsY9roFuPaMre92g1/DZBoLU2BXFpdmnhJXf1YaDHVuBIITP0AhtQfZpu8KNXm/Cmg==
+"@roadiehq/backstage-plugin-github-insights@^1.1.8":
+  version "1.1.8"
+  resolved "https://registry.npmjs.org/@roadiehq/backstage-plugin-github-insights/-/backstage-plugin-github-insights-1.1.8.tgz#d2a46ee80f9529c75e7bdf58b214b59c6e8b1eda"
+  integrity sha512-0Yag3yKe3QjhcFL00+krzPmcrmXqCBZStn56IXYjVOuz2QtSIP0rYIeOQwNFKkTN6VFGTGWS9krdnDBynkptAQ==
   dependencies:
-    "@backstage/catalog-model" "^0.7.4"
-    "@backstage/core" "^0.7.3"
-    "@backstage/plugin-catalog-react" "^0.1.4"
-    "@backstage/theme" "^0.2.5"
+    "@backstage/catalog-model" "^0.7.7"
+    "@backstage/core" "^0.7.8"
+    "@backstage/plugin-catalog-react" "^0.1.5"
+    "@backstage/theme" "^0.2.7"
     "@date-io/core" "2.10.7"
     "@material-ui/core" "^4.11.0"
     "@material-ui/icons" "^4.9.1"
-    "@material-ui/lab" "4.0.0-alpha.57"
-    "@octokit/rest" "^18.1.1"
-    "@octokit/types" "^6.13.0"
+    "@material-ui/lab" "4.0.0-alpha.45"
+    "@octokit/rest" "^18.5.3"
+    "@octokit/types" "^6.14.2"
     history "^5.0.0"
     react "^16.13.1"
-    react-dom "^16.13.1"
+    react-dom "^17.0.2"
```

Comment:
@Irma12  i see this still bumps to react-dom 17 while we are on 16 (as is your own react dependency above) - this one also needs to be backtracked to `^16.13.1`

Reply:
Hey @freben I backtracked it, hope it is ok now :)